requirements.txt

```diff
@@ -53,6 +51,8 @@ sseclient==0.0.19
 stevedore==1.28.0
 tooz==1.62.0
 unittest2
+# TODO: Move this to into the StackStorm organization
+git+https://github.com/blag/watchdog3.git#python2.7-support
```

Comment:
This file is auto generating when you run "make requirements". I assume it was updated manually since it's missing "inotify" dependency.

Reply:
It looks like you will also need to add `watchdog3` linux pack requirement to `st2common/in-requirements.txt` otherwise it won't be included in final generated `requirements.txt`.

Please also add a comment in there why we need to do that.

Reply:
Just noticed notation is also wrong, should be ``@git_revision#python_egg_name``, so ``git+https://github.com/blag/watchdog3.git@python2.7-support#egg=watchdog3``